### PR TITLE
CM-602 add a test for each module item type

### PR DESCRIPTION
**Pull request by cguanzon**

### CODE REVIEW

File: tests/test.module-items.js [hunk 1]
```diff
@@ -15,10 +15,102 @@ test("Assignment Items work", async t => {
...
+  const title = `341540_440x350.jpg`
+  await t
+    .expect(Selector(`${itemCss} svg[name='IconPaperclip']`).exists)
+    .ok()
+    .click(Selector(`${itemCss} a`).withText(title))
+    .expect(Selector('h1').withText(title).exists)
+    .ok()
+});
+
+test("File Items work", async t => {
```
Comment (mauriciozaffari) [inline tests/test.module-items.js]: @cguanzon duplicate from line 43?

File: tests/test.module-items.js [hunk 2]
```diff
@@ -15,10 +15,102 @@ test("Assignment Items work", async t => {
...
+      ).withText(`1 Questions`).exists)
+    .ok()
+    .click(Selector(`${itemCss} a`).withText(title))
+    .expect(Selector('h1').withText(title).exists)
+    .ok()
+});
+
+test("File Items work", async t => {
+  const itemCss = '.ExpandCollapseList-item:nth-of-type(3)'
+  const title = `341540_440x350.jpg`
```
Comment (mauriciozaffari) [inline tests/test.module-items.js]: if theres nothing to interpolate should't we use double/single quote instead of back-tick?

**Comment by cguanzon** [comment]
@mauriciozaffari yes, we choose the squash and merge option all the time

**Comment by mauriciozaffari** [comment]
> @mauriciozaffari yes, we choose the squash and merge option all the time
 
oh, got it, thanks! :D

**Pull request merged**